tests/test_events.py

```diff
@@ -29,42 +28,45 @@ async def _ignore_test_consoleLog_logEntryAddedEventEmitted(websocket):
             "expression": "console.log('some log message')",
             "target": {"context": context_id}}})
 
+    # The order of the two responses are not guarenteed
+    resp1 = await read_JSON_message(websocket)
+    resp2 = await read_JSON_message(websocket)
```

Comment:
`log_entry_added_event_response` and `command_done_response` seem more appropriate here

Reply:
updated